.travis.yml

```diff
@@ -22,6 +22,8 @@ before_script:
 
 script:
   - bash ncbidl.sh -v -p 2 /tmp/db blast/db/vector
+  - bash ensembldl.sh -v -p 2 pub/release-93/fasta/gallus_gallus/dna/Gallus_gallus.Gallus_gallus-5.0.dna.chromosome.3 /tmp/db
+  - bash ensembldl.sh -v -p 2 pub/release-93/bed/ensembl-compara/70_mammals.gerp_constrained_element/gerp_constrained_elements.bos_taurus /tmp/db
```

Comment:
Please split the command(s) into multiple lines using `- >-` as is done with the `diamonddb.sh` test. This improves code readability.

Reply:
done